build.sbt

```diff
@@ -310,7 +310,9 @@ lazy val core = (project in file("core"))
       "-language:reflectiveCalls",
       "-unchecked",
       "-Xcheckinit",
-      "-Xlint:infer-any"
+      "-Xlint:infer-any",
+      "-Xsource:3",
+      "-Xsource-features:case-apply-copy-access"
```

Comment:
Unlike firrtl and svsim, this is going to be changing APIs that may actually affect users so we need to potentially report it in the release notes. If you remove this option, what are the errors?